docs/release-notes.txt

```diff
@@ -8,6 +8,35 @@
  Version 0.13
 **************
 
+Version 0.13.6
+==============
+
+**Release Date:** October 14, 2020
+
+**Improvements**
+
+- Agent: The ``boot_disk_source_image`` field for GCP dynamic agents and ``image_id`` field for AWS dynamic agents are now optional.
+
+- Documentation: Ship Swagger UI with Determined documentation.  The ``/swagger-ui`` endpoint has been renamed to ``/docs/rest-api``.
+
+- Kubernetes: Add support for configuring memory and CPU requirements for the Determined database when installing via the Determined Helm Chart.
+
+- Kubernetes: Add support for configuring the `storageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/>`__ that is used when deploying a database using the Determined Helm Chart.
+
+**Bug Fixes**
+
+- Harness: Do not require the master to present a full TLS certificate chain when the certificate is signed by a well-known Certificate Authority.
+
+- Harness: Fix a bug which affected ``TFKerasTrial`` using TensorFlow 2 with ``gradient_aggregation`` > 1.
+
+- Master: Fix a bug where the master instance would fail if an experiment could not be read from the database.
+
+- WebUI: Preserve the colors used for multiple metrics on the metric chart.
+
+- WebUI: Fix the ability to cancel a batch of experiments.
+
+- WebUI: Fix the ``humanReadableFloat`` bug on the Experiment Details page.
```

Comment:
We should generally avoid writing release note entries that refer to internal function names or similar internal concepts. Can you suggest some revised text, @hkang1 ?

Reply:
@neilconway Totally agree with avoiding internal functions. In this case though, the Experiment Details page failed to render and it's just a blank page, and the only distinguishing property is that there is a console error that complains about `humanReadableFloat`... Is this better? `- WebUI: Fix the issue of Experiment Details page not rendering due to a ``humanReadableFloat`` error.`

Reply:
In what situation does the error arise?

Reply:
ah that's a great way to look at it. How about this:
```
- WebUI: Fix the issue of the Experiment Details page not rendering when the latest validation metric is not available.
```

Reply:
Looks great! Thanks @hkang1